.gitignore

```diff
@@ -77,3 +77,10 @@ default.db
 
 # Geckodriver for Jasmine
 geckodriver/
+
+# Static assets
+enterprise/assets/
```

Comment:
Isn't this where Webpack is going to put the built assets? We want those in version control, right?

Reply:
Webpack actually only generates the `enterprise/static/enterprise/bundles` folder. This `assets` folder is generated by `python manage.py collectstatic --noinput` inside of the `static` make target. In the future we'll want to be keeping all of these generated files out of version control as they're only necessary while provisioning. Since we're installed through `pip` right now because we aren't a service, I'm only keeping the `bundles` folder around as a place to grab CSS inside of our templates. Eventually we'd ignore even this and pick up assets through the Webpack loader.